files_in_root.py

```diff
@@ -0,0 +1,6 @@
+def dir():
+    import os
```

Comment:
Typically imports writes at the beginning of the file. Before any declaration

Reply:
Ok, i'll change their places in new version